axelor-account/src/main/resources/views/Invoice.xml

```diff
@@ -1340,7 +1340,19 @@ InvoiceSet.length > 0))"
 	</action-view>
 
 	<search-filters name="customer-invoices-filters" model="com.axelor.apps.account.db.Invoice" title="Customer invoices filters">
-	    <field name="address" hidden="true"/>
+		<field name="company" hidden="true" if="!__config__.app.getApp('base').getEnableMultiCompany()"/>
+		<field name="tradingName" hidden="true" if="!__config__.app.getApp('base').enableTradingNamesManagement"/>
+		<field name="pfpValidateStatusSelect" hidden="true" if="!__config__.app.getApp('account').getActivatePassedForPayment()"/>
+	    <field name="pfpValidatorUser" hidden="true" if="!__config__.app.getApp('account').getActivatePassedForPayment()"/>
+	    <field name="project" hidden="true" if="!__config__.app.getApp('business-project').getGenerateProjectOrder()"/>
```

Comment:
Here first condition should be __config__.app.isApp('business-project'), as account module does not depends on it